src/core/tsi/ssl_transport_security.cc

```diff
@@ -1047,18 +1070,25 @@ static tsi_result ssl_handshaker_result_extract_peer(
     SSL_get0_next_proto_negotiated(impl->ssl, &alpn_selected,
                                    &alpn_selected_len);
   }
-
+  STACK_OF(X509) *peer_chain = SSL_get_peer_cert_chain(impl->ssl);
```

Comment:
Nit. Could you please add some comments about `SSL_get_peer_cert_chain` API w.r.t. its different behavior at client and server side? That is, `peer_chain` will not include a leaf certificate if the API is called at the server side.

Reply:
Added